tests/test_update_api_key.py

```
@@ -366,6 +342,77 @@ def test_update_api_key_skips_blank_env_lines(self, tmp_path, fake_crypto):
+        """Rotation should back up mutable files when present."""
+        env_file = tmp_path / ".cursor" / ".env"
+        env_file.parent.mkdir(parents=True, exist_ok=True)
+        env_file.write_text("OPENAI_API_KEY=encrypted:old\n")
+
+        with patch("update_api_key.Path.home", return_value=tmp_path):
+            update_api_key.rotate_api_key("sk-" + "r" * 40)
+
+        backups = list(env_file.parent.glob(".env.bak.*"))
+        assert backups, "Expected rotation to create at least one backup file"
+
+    def test_batch_update_from_file_multiple_profiles(self, tmp_path, fake_crypto):
```

Ruff (0.13.2)

Unused method argument: `fake_crypto`

(ARG002)